src/theme/palette.js

```diff
@@ -1,52 +1,61 @@
 import { colors } from '@material-ui/core'
-
+import { EOSCR_THEME } from '@eoscostarica/eoscr-theme'
```

Comment:
Move this import to the top on the file, remember the order of imports and also add a blank space before variables